bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/Brick.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api;
+
+public interface Brick {
+
+	Area area();
+
+	Position position();
+
+	String getLabel();
```

Comment:
We discussed that, better to not intoduce this property - anyway it will be insufficient for us